### DISCUSSION TITLE
SpeedAnglePair... pair

### DISCUSSION THREAD

**Discussion by GalexY727** [Patribots4738/Crescendo2024#42] [Q&A] [answered]
With the shooter being designed to give the note spin, there is a left and right motor as seen in the subsystem. We know we will need to apply spin _somehow,_  but not quite how yet. Ideally, there would be a ratio/static offset between the wheels, but if there isn't how should we manage that in the code? Should SpeedAnglePairs have a Pair for the speeds... like a Speed**s**Angle pair? Or should it be like a SpeedsAngleTriplet, or something else?

**Comment by RudyG252** [comment, marked as answer]
I think we should try having a set offset (doesn't have to be linear) between two wheels first and if that's inconsistent we can try triplet or something similar.

**Comment by jacueblol** [comment]
If the offset doesn't work, another option is to create another class inside the speed angle pair, a class of the left and right motor speeds. This makes the speed angle pair unchanged except for the new param in the pace of the speeds for the rightAndLeftSpeeds class or something like that.

**Comment by jacueblol** [comment]
just closing some of the very old discussions
